modules/abi-oci-resources-services.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_oci/installing-oci-agent-based-installer.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="abi-oci-resources-services_{context}"]
+= Creating {oci} infrastructure resources and services
+
+You must create an {oci} environment on your virtual machine (VM) shape. By creating this environment, you can install {product-title} and deploy a cluster on infrastructure that supports a wide range of cloud options and strong security policies. Having prior knowledge of {oci} components can help you with understanding the concept of {oci} resources and how you can configure them to meet your organizational needs.
```

Comment:
Could it be? "deploy a cluster on `an` infrastructure" or "deploy a cluster on `any` infrastructure"